lib/schedule/health/checkers/routes_checker.ex

```diff
@@ -11,6 +12,13 @@ defmodule Schedule.Health.Checkers.RoutesChecker do
   def healthy?(%{min_length: min_length}) do
     routes_fn = Application.get_env(:skate_web, :routes_fn, &Schedule.all_routes/0)
 
-    routes_fn.() |> length() >= min_length
+    length = routes_fn.() |> length()
+    passfail = length >= min_length
```

Comment:
One common idiom in Elixir is to name booleans / functions that return booleans with a `?` at the end, so in this case you might choose a name like `pass?`.